docs/guides/istio.md

```diff
@@ -1,39 +1,325 @@
+        # Specify the certificate issuer for your namespace or cluster. For example:
+        # cert-manager.io/issuer: pomerium-issuer
+        kubernetes.io/ingress.class: pomerium
+        ingress.pomerium.io/pass_identity_headers: "true"
+        ingress.pomerium.io/policy: |
+          - allow:
+              or:
+                - domain:
+                    is: example.com
+      hosts:
+        - "grafana.localhost.pomerium.io"
+      tls:
+      - hosts:
+        - grafana.localhost.pomerium.io
+        secretName: grafana.localhost.pomerium.io-tls
+    persistence:
+      type: pvc
+      enabled: false
+      # storageClassName: default
+      accessModes:
+        - ReadWriteOnce
+      size: 10Gi
+      # annotations: {}
+      finalizers:
+        - kubernetes.io/pvc-protection
+    ```
+
+    ::: tip
+    Persistence is required to retain user data. Review the [Grafana Helm chart configuration](https://github.com/grafana/helm-charts/tree/main/charts/grafana#configuration) options to set the values for your environment.
+    :::
+
+1. Install Grafana to the cluster:
+
+    ```bash
+    helm upgrade --install grafana grafana/grafana --values grafana-values.yaml
+    ```
+
+1. Follow the instructions in the terminal output to log in as the admin user. Follow the [Add Users to Grafana](/guides/grafana.md#add-users-to-grafana) section of our Grafana guide to add a user that can be identified by the Pomerium JWT.
+
+
+1. To the same file, add the following values to the `grafana.ini` section.
+
+    <<< @/examples/kubernetes/istio/grafana.ini.yml
+
+    This tells Grafana to use the email address provided in the `X-Pomerium-Jwt-Assertion` JWT and associate it with the matching Grafana user. It also disabled Grafana's login form. See Grafana's [JWT authentication](https://grafana.com/docs/grafana/latest/auth/jwt/) documentation for more configuration options.
+
+1. Upgrade Grafana with the new configuration options:
+
+    ```bash
+    helm upgrade --install grafana grafana/grafana --values grafana-values.yaml
+    ```
+
+1. Now when you visit the Grafana route, you should be signed in as the user matching your Pomerium claim. To finalize the installation, create a new `grafana-istio-policy.yaml` file. Adjust the matchers and host values for Grafana, and enable `forwardOriginalToken`:
+
+    ```yaml{15}
+    apiVersion: security.istio.io/v1beta1
+    kind: RequestAuthentication
+    metadata:
+      name: grafana-require-pomerium-jwt
+    spec:
+      selector:
+        matchLabels:
+          app.kubernetes.io/name: grafana # This matches the label applied to our test service
+      jwtRules:
+      - issuer: "authenticate.localhost.pomerium.io" # Adjust to match your Authenticate service URL
+        audiences:
+          - grafana.localhost.pomerium.io # This should match the value of spec.host in the services Ingress
+        fromHeaders:
+          - name: "X-Pomerium-Jwt-Assertion"
+        forwardOriginalToken: true
```

Comment:
I don't know why but this is outlined in a black bar in the deploy preview. Just pointing it out

Reply:
Because I'm highlighting this line, noted on L275 with `{15}` as being the important change compared to the previous example of this sort of file above.